mchf-eclipse/hardware/mchf_board.h

```diff
@@ -1335,7 +1335,7 @@ typedef struct TransceiverState
 
 	uchar	scope_trace_colour;	// color of spectrum scope trace;
 	uchar	scope_grid_colour;	// saved color of spectrum scope grid;
-	ulong	scope_grid_colour_active;	// active color of spectrum scope grid;
+	ushort	scope_grid_colour_active;	// active color of spectrum scope grid;
```

Comment:
It seemed to be inconsitent with `#define RGB(red,green,blue)(uint16_t)(((red>>3)<<11)|((green>>2)<<5)|(blue>> 3))
`